src/openfermion/circuits/low_rank_test.py

```diff
@@ -294,10 +294,10 @@ def test_one_body_square_decomposition(self):
 
             # Get the squared one-body operator via one-body decomposition.
             if abs(eigenvalues[l]) < 1e-6:
-                with self.assertRaises(ValueError): # pragma: no cover
-                    prepare_one_body_squared_evolution( # pragma: no cover
-                        one_body_squares[l]) # pragma: no cover
-                continue # pragma: no cover
+                with self.assertRaises(ValueError):  # pragma: no cover
```

Comment:
Maybe leave a comment on why this is no cover

Reply:
Maybe set low-rank decomp to threshold zero to go through this pathway?

Reply:
Yep, this works - good catch!